packages/agents/setup.py

```diff
@@ -2,10 +2,11 @@
 
```

Comment:
It's nice to have a `setup.py` file for a project. However, in our case do we need that if nowhere in a project setup we are actually using it? There isn't a single `pip install .` or `pip install -e .` for working on a project instruction. So, (also keeping in mind that timeline is a bit tide) my suggestion would be to remove that since it's for now just introducing a noise (ish) vibe to a repo. If you guys are still eager to keep it I think several changes need to be made to make pip installation process slightly more correct. Like name should be the same as the package it installs and so on...

Reply:
I also was wondering why it was added in the first place and I agree with you. @L0czek wdyt? Do we really need spade_agents as an installable module?

Reply:
Got rid of unnecessary clutter to make it work with a shared `constants` package.

Reply:
sure we might get rid of it, I think I added it when we the docker support wasn't complete and I was struggling with my python 3.10 instalation and conda. Yes I agree that now as we run this in docker we might get rid of it.

Reply:
No problem! Thanks! :)